src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -143,8 +141,9 @@ public async Task<IReadOnlyCollection<object>> CreateDescriptionAsync(Cancellati
                 block.Inlines.Add(run);
             }
 
-            // Encapsulates the textblock within a border. Gets foreground/background colors from the options menu.
+            block.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
```

Comment:
While it's good to measure only what's needed, I imagine this won't be much of a perf gain. TextBlock measurement is the expensive part, the border likely had very little overhead here.

Reply:
Yes, this was an opportunistic improvement from the performance trace. The lazy evaluation is the big win, and this is the distant second but easy to fix.